man/coredns-rewrite.7

```diff
@@ -18,7 +18,7 @@ A simplified/easy-to-digest syntax for \fIrewrite\fP is...
 .RS
 
 .nf
-rewrite [continue|stop] FIELD [FROM TO|FROM TTL]
+rewrite [continue|stop] FIELD [TYPE] [(FROM TO)|TTL] [OPTIONS]
```

Comment:
revert any changes to this file, will be done automatically from a github workflow